shared/actions/tracker.js

```diff
@@ -588,6 +557,9 @@ function serverCallMap (dispatch: Dispatch, getState: Function, skipPopups: bool
     'keybase.1.identifyUi.displayUserCard': ({card}, response) => {
       response.result()
       requestIdle(() => {
+        if (isGetProfile) { // cache profile calls
```

Comment:
that's fine right, because if we make another request it won't do this work because this is pending.